app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/triage/singledisposals/completion_date.scala.html

```diff
@@ -82,7 +82,14 @@
    monthValue = form.data.get("completionDate-month"),
    yearValue = form.data.get("completionDate-year"),
    helpText = Some(Html(messages(s"$key$userKey.helpText"))),
-   errorKey = errorKey.map(e => s"${e.key}.${e.message}"),
+   errorKey = errorKey.map(e => {
+        if(e.message == "error.tooFarInPastWithArg") {
+            messages(s"${e.key}.${e.message}", s"${e.args(0)}")
+        } else {
+            s"${e.key}.${e.message}"
+        }
+
+        }),
```

Comment:
we can use only one statement like 

`errorKey = errorKey.map(e => messages(s"${e.key}.${e.message}", s"${e.args(0)}")),`